build.sbt

```diff
@@ -149,7 +148,18 @@ lazy val server = project
     Universal / packageBin := (Universal / packageBin)
       .dependsOn(Assets / WebKeys.assets)
       .value,
-    reStart := reStart.dependsOn(Assets / WebKeys.assets).evaluated,
+    Test / test := {
+      startElasticsearch.value
+      (Test / test).value
+    },
+    reStart := {
+      startElasticsearch.value
+      reStart.evaluated
```

Comment:
Should we wait for elasticsearch to be ready before we invoke reStart?

Reply:
`startElasticsearch` only checks that the http GET request replies with OK. But then the application waits for elastic to be ready using  the health api.

Reply:
Here, `startElasticsearch` and `reStart` will be executed in parallel. So, I think it’s possible that sometimes the application tries to reach the health API before elasticsearch is even reachable. I think we should consider using `Def.sequential` here.

Reply:
Oh yes that's right. Thanks for catching it.